test/hl/multi_benchmark.py

```diff
@@ -249,7 +250,13 @@ def run_benchmark(test_name, test_func, stats, datasets, num_iters):
     dt = np.int32
     stats = {}
 
-    fs = [h5py.File("/home/test_user1/h5pyd_multi_bm_" + str(i), mode='w') for i in range(count)]
+    fs = []
+
+    for i in range(count):
+        filename = getTestFileName(f"bm_{i:04d}", subfolder="multi_bm")
+        f = h5py.File(filename, mode='w')
```

Comment:
If the user hasn't specifically created the `multi_bm`, this will fail. We should either create that here or check for its existence to throw a helpful error